gsd-web/quasar.conf.js

```diff
@@ -9,6 +9,7 @@
 /* eslint-env node */
 const ESLintPlugin = require('eslint-webpack-plugin')
 const { configure } = require('quasar/wrappers');
+const envConfig = require('./src/boot/env');
```

Comment:
Requiring one of the boot loaders within the configuration file caught my eye - I don't know enough about how NodeJS handles its boot sequence and dependencies to know if this has the potential to create weird edge cases or not.

It doesn't look like we actually inject the new `env` boot file into the boot loading sequence, so I suspect this file should actually live somewhere else in the code. (where exactly, I'm not sure).